### PR TITLE
[#10] Add HTTP feeder library and a test for /health

### PR DESCRIPTION
**Pull request by coffeecup-winner**

### CODE REVIEW

File: src/KawaBot.Feeder/KawaBot.Feeder.fsproj [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>81ffe185-aa34-428e-941d-0718ef968d76</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <RootNamespace>KawaBot.Feeder</RootNamespace>
+    <AssemblyName>KawaBot.Feeder</AssemblyName>
+    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
```
Comment (sirIrishman) [inline src/KawaBot.Feeder/KawaBot.Feeder.fsproj]: Lets use the 4.6.2 version for all projects?

File: src/KawaBot.Tests/KawaBot.Tests.fsproj [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>d5b7130d-7ece-4d8a-91e1-ed7a4e6b444c</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <RootNamespace>KawaBot.Tests</RootNamespace>
+    <AssemblyName>KawaBot.Tests</AssemblyName>
+    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
```
Comment (sirIrishman) [inline src/KawaBot.Tests/KawaBot.Tests.fsproj]: Lets use the 4.6.2 version for all projects?

File: src/KawaBot.Feeder/KawaBot.Feeder.fsproj (hunk 1 above)
Comment (coffeecup-winner) [inline src/KawaBot.Feeder/KawaBot.Feeder.fsproj]: done

File: src/KawaBot.Tests/KawaBot.Tests.fsproj (hunk 2 above)
Comment (coffeecup-winner) [inline src/KawaBot.Tests/KawaBot.Tests.fsproj]: done

**Pull request merged**